lms/djangoapps/instructor_task/tasks.py

```diff
@@ -199,3 +202,14 @@ def get_student_forums_usage(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('generated')
     task_fn = partial(push_student_forums_data_to_s3, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+
+# pylint: disable=not-callable
+@task(base=BaseInstructorTask, routing_key=settings.ORA2_RESPONSES_DOWNLOAD_ROUTING_KEY)
+def get_ora2_responses_email(entry_id, xmodule_instance_args):
+    """
+    Generate a CSV of ora2 responses and push it to S3.
```

Comment:
If you take my advice above and go for a parameterized endpoint instead, though, we won't need this duplicate method.